src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -4,12 +4,31 @@
 RSpec.describe Webui::ObsFactory::StagingProjectsController, type: :controller, vcr: true do
   render_views
 
+  let(:description) do
+    <<-DESCRIPTION
+      requests:
+        - { id: #{declined_bs_request.number} }
+    DESCRIPTION
+  end
   let(:factory) { create(:project, name: 'openSUSE:Factory') }
-  let!(:factory_staging_a) { create(:project, name: 'openSUSE:Factory:Staging:A', description: 'Factory staging project A') }
+  let!(:factory_staging) { create(:project, name: 'openSUSE:Factory:Staging') }
+  let!(:factory_staging_a) { create(:project, name: 'openSUSE:Factory:Staging:A', description: description) }
```

Comment:
`factory_staging_a` and `factory_staging_b` don't seem to be used directly. They should be moved in a `before` block.

Reply:
Good point. I'll change it